go/internal/pkg/flags/flags.go

```diff
@@ -7,7 +7,12 @@ var (
 	Gateway = &cli.StringFlag{
 		Name:  "gateway",
 		Usage: "gateway websocket connection string",
-		Value: "ws://127.0.0.1:28333/ws",
+		Value: "ws://127.0.0.1:28346/ws",
```

Comment:
why change the default ?

Reply:
because of my local gateway.. will change